client/ayon_maya/plugins/publish/collect_fbx_animation.py

```diff
@@ -34,3 +42,28 @@ def process(self, instance):
                 ))
             if skeleton_content:
                 instance.data["animated_skeleton"] = skeleton_content
+
+        attribute_values = self.get_attr_values_from_data(
+            instance.data
+        )
+
+        instance.data["upAxis"] = attribute_values.get(
+            "upAxis", self.up_axis)
+        instance.data["inputConnections"] = attribute_values.get(
+            "inputConnections", self.input_connections)
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = super(CollectFbxAnimation, cls).get_attribute_defs()
+        defs.extend([
+            UISeparatorDef("sep_fbx_options"),
```

Comment:
Does this mean that the `optional` checkbox is outside of this group? Is that expected?